frontend/app/src/i18n/en.json

```diff
@@ -100,6 +100,7 @@
     "addedBy": "{changedBy} added {changed} to the {level}",
     "admin": "admin",
     "advanced": "Advanced",
+    "airdropWarning": "In order to qualify for the next Airdrop, you must be a member of the **{channelName}** channel in the **{communityName}** community. [Click here to join.]({url})",
```

Comment:
"In order to qualify for the next Airdrop" should probably say "In order to qualify for the next "CHIT for CHAT" airdrop". Because otherwise just saying "... the next Airdrop" is ambiguous.

Reply:
I don't really think that's necessary. It's already there in the community name and I don't think there is any doubt about what it means. And I don't want the message to be any longer than it needs to be.

Reply:
Oh yeah it'll be in the community name